app/helpers/application_helper.rb

```diff
@@ -15,4 +15,8 @@ def style_error_class(resource, field, css)
         return css.to_s
       end
 
+      def subscribers
+        Subscription.where(account_id: current_account.id)
+      end
```

Comment:
The right place for logics like this would be controllers i think, can we move it to the application controller instead?